nautobot/core/ui/object_detail.py

```diff
@@ -1176,6 +1176,7 @@ def __init__(
         context_object_key=None,
         ignore_nonexistent_fields=False,
         label=None,
+        key_transforms=None,
```

Comment:
This should probably be implemented on `KeyValueTablePanel` parent class instead of on `ObjectFieldsPanel`, matching `value_transforms` functionality.